vercel.json

```diff
@@ -40,7 +40,7 @@
         },
         {
           "key": "Content-Security-Policy",
-          "value": "font-src 'self' fonts.gstatic.com gstatic.com *.gstatic.com;img-src 'self' data: *.arcana.network;script-src 'self' 'unsafe-inline' js.stripe.com *.google-analytics.com *.googletagmanager.com network.us1.list-manage.com;style-src 'self' 'unsafe-inline' https://fonts.googleapis.com;frame-src js.stripe.com *.arcana.network;frame-ancestors 'self';"
+          "value": "font-src 'self' fonts.gstatic.com gstatic.com *.gstatic.com;img-src 'self' data: *.arcana.network;script-src 'self' 'unsafe-inline' js.stripe.com *.google-analytics.com *.googletagmanager.com network.us1.list-manage.com;style-src 'self' 'unsafe-inline' https://fonts.googleapis.com;frame-src js.stripe.com *.arcana.network;frame-ancestors 'self'; *.vercel.app"
```

Comment:
Don't allow *.vercel.app in csp as it is literally unsafe

Reply:
[reverted](https://github.com/arcana-network/developer-dashboard/pull/381/commits/8757d59a5edc3861ec07db9e9fd93967b278b519)